lib/datadog/ci/ext/environment.rb

```diff
@@ -213,16 +213,19 @@ def extract_github_actions(env)
           ref = env['GITHUB_REF'] if ref.nil? || ref.empty?
           branch, tag = branch_or_tag(ref)
 
+          pipeline_url = "#{env['GITHUB_SERVER_URL']}/#{env['GITHUB_REPOSITORY']}/actions/runs/#{env['GITHUB_RUN_ID']}"
+          pipeline_url = "#{pipeline_url}/attempts/#{env['GITHUB_RUN_ATTEMPT']}" if env['GITHUB_RUN_ATTEMPT']
```

Comment:
Is it just me, or we don't have any test coverage for cases where there's no `GITHUB_RUN_ATTEMPT`?

Reply:
the first element in the fixture array should cover just that: https://github.com/DataDog/dd-trace-rb/pull/1771/files#diff-456097a8e01e4ac5fd02248c6322a3af6b902170c1c321635f71981542d14bf2R19

Reply:
Totally missed it 😅